### DISCUSSION TITLE
PHP Fatal error:  require(): Failed opening required 'Symfony/Component/Dotenv/autoload.php' (include_path='.:/usr/share/php') in /srv/nominatim/Nominatim-3.7.1/lib-php/dotenv_loader.php on line 3

### DISCUSSION THREAD

**Discussion by raynerhoward** [osm-search/Nominatim#2308] [Q&A]
**Describe the bug**

After entering in the following command as the nominatim (or postgres) user:
/srv/nominatim/build/utils/setup.php --osm-file /home/wrecker/planet_-85.997,34.825_-84.615,35.617.osm.pbf --all 2>&1 | tee setup.log

I get an error:
PHP Warning:  require(Symfony/Component/Dotenv/autoload.php): failed to open stream: No such file or directory in /srv/nominatim/Nominatim-3.7.1/lib-php/dotenv_loader.php on line 3
PHP Fatal error:  require(): Failed opening required 'Symfony/Component/Dotenv/autoload.php' (include_path='.:/usr/share/php') in /srv/nominatim/Nominatim-3.7.1/lib-php/dotenv_loader.php on line 3

**To Reproduce**

cd /srv/nominatim/build/
./utils/setup.php --osm-file /home/wrecker/planet_-85.997,34.825_-84.615,35.617.osm.pbf --all 2>&1 | tee setup.log

**Software Environment (please complete the following information):**
- Nominatim version: 3.7.1
- Postgresql version: 12
- Postgis version: POSTGIS="3.0.0 r17983" [EXTENSION] PGSQL="120" GEOS="3.8.0-CAPI-1.13.1 " PROJ="6.3.1" LIBXML="2.9.4" LIBJSON="0.13.1" LIBPROTOBUF="1.3.3" WAGYU="0.4.3 (Internal)"

- OS: Linux Mint 20.1 running on latest VirtualBox

**Hardware Configuration (please complete the following information):**
- RAM: 4GB
- number of CPUs: 2
- type and size of disks: SSD
- bare metal/AWS/other cloud service: VirtualBox Version 6.1.22 r144080 (Qt5.6.2)

**Postgresql Configuration:**

**Additional context**

**Comment by lonvia** [comment]
Either install [Symfony dotenv](https://symfony.com/doc/3.4/components/dotenv.html) or switch to using the `nominatim` tool as per current installation instructions.

**Reply by raynerhoward** [reply]
Good docs.

nominatim import --osm-file <data file> 2>&1 | tee setup.log

from https://nominatim.org/release-docs/latest/admin/Import/#initial-import-of-the-data
